app/components/meetings/sessions/files-view.vue

```diff
@@ -4,9 +4,10 @@
     <!-- Medium view and above -->
     <div class="document-files">
         <div v-for=" {language, text, contentType, url, public: isPublic, allowPublic, _id} in files" v-bind:key="_id" >
-
           <span class="d-none d-md-inline">
-            <i :style="{ visibility: (text?'visible':'hidden') }" class="fa fa-file-text-o" aria-hidden="true" @click="showPreview(text, language)"></i>
+            <a @click="showPreview(text, language)" :style="{ visibility: (text?'visible':'hidden') }" :class="{ 'btn btn-lg btn-outline-dark': showPreviewAsButton}">
+              <i :style="{ visibility: (text?'visible':'hidden') }" class="fa fa-file-text-o" aria-hidden="true"></i>
```

Comment:
no need to have the visibility style here has it was moved above
```suggestion
              <i class="fa fa-file-text-o" aria-hidden="true"></i>
```